order/client_test.go

```diff
@@ -19,7 +19,13 @@ func TestOrderGet(t *testing.T) {
 }
 
 func TestOrderList(t *testing.T) {
-	i := List(&stripe.OrderListParams{})
+	i := List(&stripe.OrderListParams{
+		StatusTransitions: &stripe.StatusTransitionsFilterParams{
+			PaidRange: &stripe.RangeQueryParams{
+				GreaterThan: int64(123456),
+			},
+		},
```

Comment:
I don't think adding the new predicate is strictly needed here because both the `form` annotations and `RangeQueryParams` serialization are pretty well tested already (see `form/form_test.go` and `params_test.go`).

Reply:
Makes sense. Removed.